docs/subnets/how-to-delete-subnet.md

```diff
@@ -0,0 +1,5 @@
+# How to Delete a Subnet Configuration
+
+To delete a created Subnet configuration, run
+
+`avalanche subnet delete <subnetName>`
```

Comment:
maybe another section on how to delete a locally deployed subnet (ie clean), and...
maybe a comment over that it is not posible to delete subnets on mainnet/fuji and this only deletes configs on CLI?